### ISSUE TITLE
Contributor guidelines should suggest communication first, not last

### ISSUE THREAD

**Issue by rmol** [freedomofpress/securedrop#5681]
## Description

In the contributor guidelines, we wait until the [very end of the document to suggest that an issue be filed before starting a PR](https://docs.securedrop.org/en/latest/development/contributor_guidelines.html#other-tips). (Our top priority, apparently, is signing commits. :laughing:)

We should ask contributors to check in with us before starting, particularly if the change will be large or hasn't been captured in an issue, to ensure they're spending their time on work that's likely to be readily accepted. It's discouraging to have PRs rejected, or even just deferred so long that they require a lot of rebasing or refactoring to keep ready to merge.

**Comment by eloquence** [comment]
@rmol I've put some more emphasis on communication in the contributor guide, in this section: https://docs.securedrop.org/en/latest/development/contributing.html#preparing-and-submitting-changes

The guide also mentions the code of conduct early on, now.

Do you feel that those changes are sufficient to resolve this issue, or would you like to see additional emphasis in other places?
